app/models/request.rb

```diff
@@ -0,0 +1,4 @@
+class Request < ApplicationRecord
+  validates :score, presence: true
+  validates :username, presence: true
```

Comment:
nice I wonder, if not having an actual score on our database will effect this, i guess not though